packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiClient } from '../.'
+import BigNumber from 'bignumber.js'
+
+/**
+ * Loan RPCs for DeFi Blockchain
+ */
+export class Loan {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Creates a loan scheme transaction.
+   *
+   * @param {number} minColRatio Minimum collateralization ratio
+   * @param {BigNumber} interestRate Interest rate
+   * @param {CreateLoanSchemeOptions} options
+   * @param {string} options.id Unique identifier of the loan scheme, max 8 chars
+   * @param {UTXO[]} [options.utxos = []] Specific UTXOs to spend
+   * @param {string} [options.utxos.txid] Transaction Id
+   * @param {number} [options.utxos.vout] Output number
```

Comment:
```suggestion
   * @param {string} options.utxos.txid Transaction Id
   * @param {number} options.utxos.vout Output number
```
within `UTXO` these are not optional.